docs/differences-from-v1.md

```diff
@@ -221,7 +221,7 @@ If you are integrating a private project and want to share more details, or if y
 
 In 1.x, `fossa report` supported `fossa report dependencies`, `fossa report licenses` and `fossa report attribution`.
 
-In 3.x, only `fossa report attribution --json` is reported. The information in the previously supported v1 commands is contained in the output from `attribution`.
+In 3.x, only `fossa report attribution --format json` is reported. The information in the previously supported v1 commands is contained in the output from `attribution`.
```

Comment:
maybe just remove the format flag here now that we support more than one?

Reply:
Can you clarify "remove the format flag"?

Reply:
Updated to:
> In 1.x, `fossa report` supported `fossa report dependencies`, `fossa report licenses` and `fossa report attribution`.
> In 3.x, only `fossa report attribution` is supported. The information in the previously supported v1 commands is contained in the output from `attribution`.

I think we only originally had `--json` in there because it was required.

Reply:
We should leave a note about `--format` being required as well.

Reply:
Done.
```markdown
In 1.x, `fossa report` supported `fossa report dependencies`, `fossa report licenses` and `fossa report attribution`.

In 3.x, only `fossa report attribution` is supported. The information in the previously supported v1 commands is contained in the output from `attribution`.
This command also now requires a `--format` argument: see [fossa report attribution](references/subcommands/report.md#specifying-a-report-format) for details.
```